### PR TITLE
Add remove-chat button next to chat title

### LINKED ISSUE #4435: Add option "Delete chat"
### User Story

As a user, I should be able to delete public channels or to delete 1:1 chat
### Description

*Type*: Feature
*Summary*: in Status Desktop app there is no option to leave public chat or delete chats.
In mobile:
leave

#### Expected behavior
"Delete chat" is available for Public and 1-1 chats;
"Delete" option for Public chat also make user leave the channel

### PR DESCRIPTION
**Pull request by vitvly**
### Summary:
This PR adds "Delete chat"/"Clear history" actions in the chat header. 

Currently not implemented via menu due to https://github.com/status-im/react-native-desktop/issues/256

Alert design relies on built-in Alert component for now.

Fixes #4435, #5147, #5156, #5136

### CODE REVIEW

**Review by vkjr — APPROVED** [review]

**Review by maxhora — APPROVED** [review]

**Review by rcullito — APPROVED** [review]
:clap:

**Comment by vitvly** [comment]
Related to https://github.com/status-im/react-native-desktop/issues/256

**Comment by vkjr** [comment]
@siphiuel, can we merge this?

**Comment by vkjr** [comment]
@siphiuel, is it blocked by https://github.com/status-im/react-native-desktop/issues/256 ?

**Review by goranjovic — APPROVED** [review]

**Comment by churik** [comment]
@siphiuel 
1) Please, get at least 1 approve from core mobile team member
2) Is it still WIP?
3) if it'll ready - we cannot merge it without design input OR status-im/react-native-desktop#256  fix (added info to #4435).
Now "Delete chat" looks like `...`:
deletechat
cc @EugeOrtiz 
IMO as we need to add functionality "Clear history" (like in mobile) - we should fix  status-im/react-native-desktop#256 first and then implement these additional features in popup menus, as it supposed to be.

File: src/status_im/ui/screens/desktop/main/chat/styles.cljs [hunk 1]
```diff
@@ -96,4 +97,30 @@
...
+   :background-color :white})
+
+(def chat-icon
+  {:width         34
+   :border-radius 34
+   :height        34
+   :margin-right  12})
+
+(defn topic-image [color]
+  (merge chat-icon
```
Comment (cammellos) [inline src/status_im/ui/screens/desktop/main/chat/styles.cljs]: you can use `assoc` here

**Comment by vitvly** [comment]
@churik I've opted to text labels in red in the right-hand side of chat header for now, similarly to how @rcullito implemented "View profile" action in https://github.com/status-im/status-react/pull/4893

**Review by cammellos — APPROVED** [review]

File: env/dev/env/config.cljs [hunk 2]
```diff
@@ -1,7 +1,5 @@
 (ns env.config)
```
Comment (cammellos) [inline env/dev/env/config.cljs:2]: This file doesn't need to be commited, not sure actually why is not in `gitignore`, does anyone know?

Comment (vitvly) [reply env/dev/env/config.cljs:2]: @cammellos this got committed as it was modified by `lein cljfmt fix`. Should I remove it from PR?

**Comment by churik** [comment]
#5136 is fixed; 

- [ ] 1) "Add to contacts" button doesn't work

The button is presented, but nothing happens on click
add_to

- [ ] 2) The app is unusable after deleting chat or clearing history

*Steps:*
2.1
 - Sign Up
 - Start 1-1 chat
 - Delete chat
 - Go to Profile
2.2
 - Sign Up
 - Start 1-1 chat
 - Clear Chat
 - open View Profile
*Expected result:* 
2.1 chat is deleted, chat list is reordered
2.2 User profile is shown
*Actual result: * 
image
*Video:* http://take.ms/M8ZY1w
*Log:* [StatusIm.txt](https://github.com/status-im/status-react/files/2201826/StatusIm.txt)

- [ ] 3) User photos are mixed in chat header

*Steps:*
 - Sign Up
 - Start 1-1 chat
 - Start second 1-1 chat with user with custom photo
 - Switch between them
*Expected result:*  chat header photo  is same as user profile photo
*Actual result: * 
image

*Video:* http://take.ms/4ET2P

- [ ] 4) During chat switching, I still face with behavior that username is cut( [video](http://take.ms/hX0CYU))
chat_header
The issue is disappearing after switching to Profile and back.
Same thing with the unread badge - until you are on chat list and receiving 10 messages in one chat - you can see only 9, but `9+` (as expected) appears after switching to Profile and back.
@siphiuel I suppose it is RN-related, should we create new issues in RN, or can reference this behavior with existing?

**Comment by churik** [comment]
@siphiuel can you take a look, does it resolve issue 2?

**Comment by vitvly** [comment]
@churik seems yes, tried clearing history/deleting chats, UI is fine.

**Comment by churik** [comment]
@siphiuel 
OK, what about 3 and 4 issues? are they also RN-related?

**Comment by maxhora** [comment]
@siphiuel @churik I will take a look at 4th issue.
Regarding 3rd issue, not sure if it's RN related or the source of image is not just get updated on active chat switch

**Comment by maxhora** [comment]
@churik I can't repro 4th issue with chat header names displaying

**Comment by churik** [comment]
@MaxRis 
it is reproducible on build https://jenkins.status.im/job/status-react/job/desktop/job/pull-requests/view/change-requests/job/PR-4636/18/ when you are switching between public/1-1 chats several times (5+) in a row

**Pull request closed without merging**